### DISCUSSION TITLE
Sonoff Temp & Humidity sensor stopped working (SNZB-02)

### DISCUSSION THREAD

**Discussion by r4nd0mbr1ck** [Koenkk/zigbee2mqtt#9392] [Devices]
So my sonoff temperature sensor suddenly stopped working and hasn't been reporting any temperature changes. It was originally connected through a router (an IKEA Tradfri outlet). Any ideas on how to debug this?

**What have I tried so far?** 

1) Moved sensor closer to the controller (Sonoff cc2531-usb-dongle)

2) Tried reconfigure via the MQTT topic. Resulted in below error 

`debug 2021-10-29 15:10:28: Received MQTT message on 'zigbee2mqtt/bridge/request/device/configure' with data '{"id": "0x00124b0023204ad4"}'
info  2021-10-29 15:10:28: Configuring '0x00124b0023204ad4'
error 2021-10-29 15:10:38: Configure failed: Error: Bind 0x00124b0023204ad4/1 msTemperatureMeasurement from '0x00124b001cce33a9/1' failed (AREQ - ZDO - bindRsp after 10000ms)
info  2021-10-29 15:10:38: Successfully configured '0x00124b0023204ad4'
info  2021-10-29 15:10:38: MQTT publish: topic 'zigbee2mqtt/bridge/response/device/configure', payload '{"data":{"id":"0x00124b0023204ad4"},"status":"ok"}'`

3) Forced remove from database using remove topic

4) Tried reconfigure again with the exact same result. 

Database.db entry for the temp sensor is as below

`{
  "id": 11,
  "type": "EndDevice",
  "ieeeAddr": "0x00124b0023204ad4",
  "nwkAddr": 10686,
  "manufId": 0,
  "manufName": "eWeLink",
  "powerSource": "Battery",
  "modelId": "TH01",
  "epList": [
    1
  ],
  "endpoints": {
    "1": {
      "profId": 260,
      "epId": 1,
      "devId": 770,
      "inClusterList": [
        0,
        3,
        1026,
        1029,
        1
      ],
      "outClusterList": [
        3
      ],
      "clusters": {
        "genBasic": {
          "attributes": {
            "modelId": "TH01",
            "manufacturerName": "eWeLink",
            "powerSource": 3,
            "zclVersion": 1,
            "appVersion": 5,
            "hwVersion": 1,
            "dateCode": "20201026"
          }
        }
      },
      "binds": [],
      "configuredReportings": [],
      "meta": {}
    }
  },
  "appVersion": 5,
  "hwVersion": 1,
  "dateCode": "20201026",
  "zclVersion": 1,
  "interviewCompleted": true,
  "meta": {
    "configured": 3
  },
  "lastSeen": 1635480823021
}`

**Comment by r4nd0mbr1ck** [comment]
Also tried resetting the sensor itself. Logs are here: https://pastebin.com/uebyvZCm

**Comment by GaWeiss** [comment]
If this is still relevant to you: Seems to be related to the issue https://github.com/Koenkk/zigbee2mqtt/issues/8971. There are some ways mentioned to fix it temporary. Maybe you give it a try.

**Comment by Tommixoft** [comment]
Not working for me too. Just stps reporting. I delete it from network readd it works once (fake temp, always the same) and never reports.
